universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -142,8 +146,24 @@ public boolean hasQuestionErrors() {
     }
 
     public ImmutableSet<ValidationErrorMessage> getQuestionErrors() {
```

Comment:
suggestion(blocking): could we add one PO box test? I know we'll add  more for #489 but it would be nice to start with one in this PR

Reply:
I don't think I have time before heading out :(

No pressure for this at all, but please feel free to add a test to this branch and merge later if it looks good!

Reply:
Tests have finally been added!